sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/grpc/FromProtoValueCodecs.java

```diff
@@ -301,6 +305,18 @@ private UUID uuidFrom(QueryOuterClass.Value value) {
     }
   }
 
+  protected static final class TimestampCodec extends FromProtoValueCodec {
+    @Override
+    public Object fromProtoValue(QueryOuterClass.Value value) {
+      return Instant.ofEpochMilli(value.getInt()).toString();
+    }
+
+    @Override
+    public JsonNode jsonNodeFrom(QueryOuterClass.Value value) {
+      return jsonNodeFactory.textNode(Instant.ofEpochMilli(value.getInt()).toString());
```

Comment:
wait is this the same as the 

`jsonNodeFactory.textNode(fromProtoValue(value));`

EDIT: OK I see this would not compile due to the `Object`.. We should have made `FromProtoValueCodec` generic

Reply:
It is possible we might only need `JsonNode` variant but originally I was not sure; for serialization either works.
I don't think generics would necessarily have helped; my experience on things like this is that they tend to cause as much issues as they solve.